packages/forklift-console-plugin/src/modules/Providers/views/create/components/OvirtProviderCreateForm.tsx

```diff
@@ -74,7 +73,7 @@ export const OvirtProviderCreateForm: React.FC<OvirtProviderCreateFormProps> = (
           type="text"
           id="url"
           name="url"
-          value={provider?.spec?.url || ''}
+          value={url}
```

Comment:
```suggestion
          value={url || ''}
```
to indicate we don't care if it's `undefined` of `''` in this `<TextInput ...>`

Reply:
We do care, `undefined` is treated as `default` while `''` is treated as `error`...

Reply:
> in this <TextInput ...>

> We do care, undefined is treated as default while '' is treated as error...

exectly, that is why i want an indication that specifically in this specific `TextInput`  we don't care and `undefined` will be translated to '', and usually in other places we do care, but in this TextInput `undefined` and '' are translating the `undefined` to '', it is an unusual behaviour different than in other places, that  is why i prefer to be verbose and explicit, here we do something different, here we translate `undefine` to ''